fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -148,6 +150,8 @@ public enum OlapTableState {
 
     private TableProperty tableProperty;
 
+    private volatile Statistics statistics = new Statistics();
```

Comment:
When fe restarted, the `statistics` has not inited and the dataSize will be 0. At this point, `createTable` behavior will not be restricted.
Should we deal with this situation?

Reply:
As we discuss, if statistics is not inited, we should try init it first.
But I think we may first implement this logic in master branch.